src/NuGet.Services.AzureSearch/DocumentUtilities.cs

```diff
@@ -0,0 +1,216 @@
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Web;
+using NuGet.Frameworks;
+using NuGet.Protocol.Catalog;
+using NuGet.Services.Entities;
+using NuGet.Services.Metadata.Catalog;
+using NuGet.Versioning;
+using PackageDependency = NuGet.Protocol.Catalog.PackageDependency;
+
+namespace NuGet.Services.AzureSearch
+{
+    internal static class DocumentUtilities
+    {
+        private static readonly DateTimeOffset UnlistedPublished = new DateTimeOffset(1900, 1, 1, 0, 0, 0, TimeSpan.Zero);
+
+        private static readonly HashSet<NuGetFramework> SpecialFrameworks = new HashSet<NuGetFramework>
+        {
+            NuGetFramework.AnyFramework,
+            NuGetFramework.AgnosticFramework,
+            NuGetFramework.UnsupportedFramework
+        };
+
+        public static void PopulateMetadata(
+            IBaseMetadataDocument document,
+            string packageId,
+            Package package)
+        {
+            document.Authors = package.FlattenedAuthors;
+            document.Copyright = package.Copyright;
+            document.Created = AssumeUtc(package.Created);
+            document.Description = package.Description;
+            document.FileSize = package.PackageFileSize;
+            document.FlattenedDependencies = package.FlattenedDependencies;
+            document.Hash = package.Hash;
+            document.HashAlgorithm = package.HashAlgorithm;
+            document.IconUrl = package.IconUrl;
+            document.Language = package.Language;
+            document.LastEdited = AssumeUtc(package.LastEdited);
+            document.LicenseUrl = package.LicenseUrl;
+            document.MinClientVersion = package.MinClientVersion;
+            document.NormalizedVersion = package.NormalizedVersion;
+            document.OriginalVersion = package.Version;
+            document.PackageId = packageId;
+            document.Prerelease = package.IsPrerelease;
+            document.ProjectUrl = package.ProjectUrl;
+            document.Published = package.Listed ? AssumeUtc(package.Published) : UnlistedPublished;
+            document.ReleaseNotes = package.ReleaseNotes;
+            document.RequiresLicenseAcceptance = package.RequiresLicenseAcceptance;
+            document.SemVerLevel = package.SemVerLevelKey;
+            document.Summary = package.Summary;
+            document.Tags = Utils.SplitTags(package.Tags ?? string.Empty);
+            document.Title = package.Title;
+        }
+
+        public static void PopulateMetadata(
+            IBaseMetadataDocument document,
+            string normalizedVersion,
+            PackageDetailsCatalogLeaf leaf)
+        {
+            document.Authors = leaf.Authors;
+            document.Copyright = leaf.Copyright;
+            document.Created = leaf.Created;
```

Comment:
Why not `AssumeUtc(…)` here, [here](https://github.com/NuGet/NuGet.Services.Metadata/pull/424/files#diff-e16587255d293a02ecfd8c0d95e68d3fR77), and [here](https://github.com/NuGet/NuGet.Services.Metadata/pull/424/files#diff-e16587255d293a02ecfd8c0d95e68d3fR85)?

Reply:
They are already `DateTimeOffset` which have specific timezone information.